graphrag/index/cli.py

```diff
@@ -168,15 +169,26 @@ def index_cli(
 
     _register_signal_handlers(progress_reporter)
 
-    outputs = asyncio.run(
-        build_index(
-            default_config,
-            run_id,
-            memprofile,
-            progress_reporter,
-            pipeline_emit,
+    if update_index_id:
+        outputs = asyncio.run(
+            update_index(
```

Comment:
This seems good for now. Once implementation starts, you may find it cleaner to fold the logic into the existing `build_index(...)` function and add a new parameter. But this will be a good placeholder for now.